robotpy_installer/installer.py

```diff
@@ -1367,6 +1378,15 @@ def install_pip(self, options):
         if len(options.requirement) != 0:
             raise NotImplementedError()
 
+        self.remote_commands.append("([ -x /usr/local/bin/pip3 ] || exit 87)")
```

Comment:
Why a subshell? We know it's bash, right?

Reply:
Allows me to ignore order of operations?

Reply:
You can still group commands with `{ ; }` though.